wordpress-20-04/files/opt/digitalocean/wp_setup.sh

```diff
@@ -106,4 +106,25 @@ wp plugin install wp-fail2ban --allow-root --path="/var/www/html"
 wp plugin activate wp-fail2ban --allow-root --path="/var/www/html"
 chown -Rf www-data.www-data /var/www/
 cp /etc/skel/.bashrc /root
+
+if [ -f "/root/.digitalocean_dbaas_password" ]; then
```

Comment:
nit: what are your thoughts on naming this something like `digitalocean_dbaas_creds`/`digitalocean_dbaas_conn`? When I see password it makes me think the file only contains the password.

Reply:
we can do that! in the boot script there is already a file named `/root/.digitalocean_password` so I figured I would follow, but that makes more sense to me

Reply:
Yeah I was almost going to say, "is it a convention I don't know of?" lol so maybe you choose

Reply:
nah im totally onboard, makes sense to me lol